src/Form/atoms/FilterOption.tsx

```diff
@@ -156,6 +159,16 @@ const FilterOption: React.FC<IFilterOption> = ({
 
   const iconWeight: number = dimensions.icons.weights['regular'];
 
+  const titleRef = useRef(null);
+  const [isOverflowing, setIsOverflowing] = useState(false);
+
+  const handleMouseEnter = () => {
+    const titleElement = titleRef.current as unknown as HTMLDivElement;
+    if (titleElement) {
+      setIsOverflowing(titleElement.scrollWidth > titleElement.clientWidth);
```

Comment:
We can avoid using state and re-rendering with below code
```
if (titleElement) {
   titleElement.title=title;
}
```

Reply:
Done, Thanks for feedback